components/board/genericlinux/board_test.go

```diff
@@ -104,8 +104,8 @@ func TestGenericLinux(t *testing.T) {
 
 	t.Run("test genericlinux gpio pin functionality", func(t *testing.T) {
 		bs, err := b.Status(ctx, nil)
-		test.That(t, err, test.ShouldBeNil)
-		test.That(t, bs, test.ShouldResemble, &commonpb.BoardStatus{})
+		test.That(t, err.Error(), test.ShouldContainSubstring, "closed")
```

Comment:
I don't get this. What's closed? Surely not the entire board component, right? Maybe it's the SPI bus named "closed"? but I don't see why you'd expect an error at all here; shouldn't status-reporting be error-free?

...TBH, I question whether this test is useful at all. If this works on any genericlinux board in practice, I don't care if it doesn't work on the mocked out version in a test. If the error is "you didn't mock out the SPI device and now you can't read from it because it doesn't exist," I'm inclined to just remove the whole paragraph.

Reply:
Rand and I discussed: my guess about the error was right, that it's saying "we tried to open the actual physical SPI device but it doesn't exist." We'll just remove the test entirely, and make sure this works on real hardware.